aspnetcore/signalr/javascript-client.md

```diff
@@ -19,38 +19,70 @@ The ASP.NET Core SignalR JavaScript client library enables developers to call se
 
 ## Install the SignalR client package
 
-The SignalR JavaScript client library is delivered as an [npm](https://www.npmjs.com/) package. If you're using Visual Studio, run `npm install` from the **Package Manager Console** while in the root folder. For Visual Studio Code, run the command from the **Integrated Terminal**.
+The SignalR JavaScript client library is delivered as an [npm](https://www.npmjs.com/) package. The following sections outline different ways to install the client library.
+
+### Install with npm
+
+If using Visual Studio, run the following commands from **Package Manager Console** while in the root folder. For Visual Studio Code, run the following commands from the **Integrated Terminal**.
 
 ::: moniker range=">= aspnetcore-3.0"
 
-  ```console
-  npm init -y
-  npm install @microsoft/signalr
-  ```
+```bash
+npm init -y
+npm install @microsoft/signalr
+```
 
 npm installs the package contents in the *node_modules\\@microsoft\signalr\dist\browser* folder. Create a new folder named *signalr* under the *wwwroot\\lib* folder. Copy the *signalr.js* file to the *wwwroot\lib\signalr* folder.
 
 ::: moniker-end
 
 ::: moniker range="< aspnetcore-3.0"
 
-  ```console
-  npm init -y
-  npm install @aspnet/signalr
-  ```
+```bash
+npm init -y
+npm install @aspnet/signalr
+```
 
 npm installs the package contents in the *node_modules\\@aspnet\signalr\dist\browser* folder. Create a new folder named *signalr* under the *wwwroot\\lib* folder. Copy the *signalr.js* file to the *wwwroot\lib\signalr* folder.
 
 ::: moniker-end
 
-## Use the SignalR JavaScript client
-
-Reference the SignalR JavaScript client in the `<script>` element.
+Reference the SignalR JavaScript client in the `<script>` element. For example:
 
 ```html
 <script src="~/lib/signalr/signalr.js"></script>
 ```
 
+### Use a Content Delivery Network (CDN)
+
+To use the client library without the npm prerequisite, reference a CDN-hosted copy of the client library. For example:
+
+```html
+<script src="https://cdnjs.cloudflare.com/ajax/libs/microsoft-signalr/3.1.3/signalr.min.js"></script>
+```
+
+The client library is available on the following CDNs:
+
+::: moniker range=">= aspnetcore-3.0"
+
+* [cdnjs](https://cdnjs.com/libraries/microsoft-signalr)
+* [jsDelivr](https://www.jsdelivr.com/package/npm/@microsoft/signalr)
+* [unpkg](https://unpkg.com/@microsoft/signalr)
```

Comment:
unpkg goes to the cjs/index.js file automatically. Maybe we should hardcode the url?
https://unpkg.com/@microsoft/signalr@next/dist/browser/signalr.min.js